react/react1/week3/homework/todo-app/src/components/TodoList.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import AddTod from "./AddTodo";
+import FancyBorder from "./FancyBorder";
+import Todo from "./Todo";
+
+export const TodoList = ({ fecthUrl }) => {
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    const timeOut = setTimeout(getData, 2000);
+    return () => clearTimeout(timeOut);

```

Comment:
Same here. You dont need to set timeout here you can call directly function here
```suggestion
    getData()
```

Reply:
Thanks, changes implemented.